Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM okteto/okteto:latest
+FROM okteto/okteto:2.18.0
```

Comment:
@jLopezbarb shall we start pinning the version for the actions? 

Should we add updating the pinned version as part of the release?

Reply:
I think we were going to use:
- main for latest
- v2 for the latest v2
- v1 for the latest v1
If we pin it we would have to change it in every CLI release and we'll need to update the actions script